docs/introduction/how-to-work-with-money.md

```diff
@@ -0,0 +1,388 @@
+
+// ...
+
+$zboziFeedProductFormBuilder->add('cpc', MultidomainType::class, [
+    'label' => $this->translator->trans('Maximum price per click'),
+    'entry_type' => MoneyType::class,
+    'required' => false,
+    'entry_options' => [
+        'currency' => 'CZK',
+        'constraints' => [
+            new MoneyRange([
+                'min' => Money::create(1),
+                'max' => Money::create(500),
+            ]),
+        ],
+    ],
+]);
+```
+
+## Money in Twig Templates
+
+Instances of the `Money` class cannot be directly converted to strings.
+To be able to display a monetary value in a template you should use one of the prepared Twig filters.
+
+These filters can be used only with an instance of `Money`.
+
+### moneyFormat
+
+Formats the amount of money as a decimal number without any currency symbol.
+Optional parameters can be provided to specify the *number of decimal places* (`null` by default, meaning all), the *decimal point character* (`"."` by default) and the *separator of thousands* (`""` by default).
+
+```twig
+{# the "money" variable contains Money::create('1234.5670') #}
+
+{{ money|moneyFormat }}                  {# renders "1234.5670" #}
+{{ money|moneyFormat(0) }}               {# renders "1235" #}
+{{ money|moneyFormat(2, ',') }}          {# renders "1234,57" #}
+{{ money|moneyFormat(null, '.', ',') }}  {# renders "1,234.5670" #}
+```
+
+### price
+
+Filter `price` formats the amount of money in a localized manner.
+It uses basic frontend currency on the current domain and the current locale (language) to format it.
+
+For example, one thousand Czech crowns would be rendered as *"CZK1,000.00"* on an English domain and as *"1 000,00 Kč"* on a Czech one.
+
+All `price*` Twig filters use this format.
+They usually differ only in the currency and locale they use.
+
+### priceText
+
+Filter `priceText` formats the amount of money in a localized manner, similarly to the `price` filter.
+The only difference is that it outputs the text *"Free"* when zero amount of money is provided.
+
+### priceTextWithCurrencyByCurrencyIdAndLocale
+
+Filter `priceTextWithCurrencyByCurrencyIdAndLocale` can be used to format the amount of money in any currency and locale.
+It outputs the text *"Free"* when zero amount of money is provided.
+
+The *currency ID* (`int`) and the *locale* (`string`) must be provided as parameters.
+
+### priceWithCurrency
+
+Filter `priceWithCurrency` can be used to format the amount of money in any currency.
+It uses the current locale (the locale of current domain or administration) to format it.
+
+The *currency* (`Currency` entity) must be provided as a parameter.
+
+### priceWithCurrencyAdmin
+
+It works similarly to `priceWithCurrency`, but it uses the default administration currency.
+
+It has no parameters.
+
+### priceWithCurrencyByCurrencyId
+
+It works similarly to `priceWithCurrency`, but it uses currency ID instead of the `Currency` entity.
```

Comment:
I think we should create some US for simplification (reduction of the amount of these filters) :thinking:

Reply:
@PetrHeinz did you create a US? Or an issue?

I'd like to resolve this conversation, but I don't know the output

Reply:
I've created US-5461 in our internal backlog.